packages/babel-plugin/src/__test__/testUtils.ts

```diff
@@ -17,10 +18,18 @@ export function babelTransform(
         },
       ],
     ],
-    filename: "test.tsx",
+    filename,
+  });
+
+  if (result === null || result.code == null)
+    throw new Error(`Could not transform`);
+
+  result = transformSync(result.code, {
+    plugins: [plugin({ types, template })],
+    filename,
```

Comment:
In actual scenarios, I believe the Kuma UI Babel plugin is loaded after reading Babel presets like React. Therefore, I have also improved the tests accordingly.